Documentation/InputSimulation/InputAnimationRecording.md

```diff
@@ -51,7 +51,42 @@ The current animation time can also be controlled directly with the [LocalTime](
 
 # Editor tools for recording and playing input animation
 
->TODO: added in separate PR
+A number of tools exist in the Unity editor for recording and examining input animation. These tools can be accessed in the _Input Recording_ window, which can be opened from the _"Mixed Reality Toolkit > Utilities > Input Recording"_ menu.
+
+<a target="_blank" href="../../Documentation/Images/InputSimulation/MRTK_InputAnimation_EditorToolsMenu.png">
+  <img src="../../Documentation/Images/InputSimulation/MRTK_InputAnimation_EditorToolsMenu.png" title="Editor tools menu" width="80%" class="center" />
+</a>
+
+> Note: Input recording and playback only works during play mode.
```

Comment:
Like the input simulation videos which were a huge hit, I have a feeling that making + sharing videos of how to use recording and playback in MRTK will be a big hit. @cre8ivepark  as FYI.

Reply:
Agreed, but we should have a stable feature set before making videos. Otherwise it's just more work if we have to redo it, and confusing when there are outdated versions.